pycheribuild/projects/cross/cheribsd.py

```diff
@@ -374,6 +374,10 @@ def _setup_cross_toolchain_config(self):
             # Building FreeBSD for RISC-V requires an external GCC:
             if self._crossCompileTarget == CrossCompileTarget.RISCV:
                 self.make_args.set(CROSS_TOOLCHAIN="riscv64-gcc")
+                # -Werror is too painful with a modern GCC...
+                self.make_args.set(NO_WERROR="YES")
+                # sys/ sets WERROR to -Werror instead
+                self.make_args.set(WERROR="")
```

Comment:
This should only be added to `BuildCHERIBSD` since upstream FreeBSD builds just fine with the GCC warnings.